sematic/plugins/abstract_storage.py

```diff
@@ -1,19 +1,16 @@
 # Standard Library
 import abc
-import enum
-from dataclasses import dataclass
-from typing import Any, Type
+from dataclasses import dataclass, field
+from typing import Dict, Optional, Type
 
-
-class PayloadType(enum.Enum):
-    URL = "URL"
-    BYTES = "BYTES"
+# Sematic
+from sematic.db.models.user import User
 
 
 @dataclass
-class ReadPayload:
-    type_: PayloadType
-    content: Any
+class Location:
+    location: str
+    headers: Dict[str, str] = field(default_factory=dict)
```

Comment:
Formulations such as `Location.location` and `read_location.location` are repetitive and not that self-explanatory.

Please rename `Location` to `StorageLocation`.

Will `location` ever not be a URL? Please rename it to `url`.

Please rename `headers` to `access_headers`.

Reply:
`location` is always a URL.